### PR TITLE
Bump apko to pick up env fix

### PR DESCRIPTION
**Pull request by jonjohnsonjr**
Fixes https://github.com/chainguard-dev/terraform-provider-apko/issues/181

### CODE REVIEW

**Comment by mattmoor** [comment]
@jonjohnsonjr are you expecting a digest change? 🤔

**Comment by jonjohnsonjr** [comment]
Hmm...

**Comment by jonjohnsonjr** [comment]
It pulled in a version of go-apk that did affect things, downgraded that... let's see.

File: go.mod [hunk 1]
```diff
@@ -5,9 +5,9 @@ go 1.21
 toolchain go1.21.0
 
 require (
-	chainguard.dev/apko v0.10.1-0.20230906171757-7c910a4b75ca
+	chainguard.dev/apko v0.10.1-0.20231017165019-27c6d86b09e6
```
Comment (imjasonh) [inline go.mod:8]: Can we bump to 0.11 and cut 0.11 of tf-apko?

Comment (jonjohnsonjr) [reply go.mod:8]: I'm lazily avoiding the digest change but we can do that.

Comment (jonjohnsonjr) [reply go.mod:8]: I don't think I can without the digest change for some reason.

**Pull request merged**